bbb-common-message/src/main/scala/org/bigbluebutton/common2/msgs/PresentationPodsMsgs.scala

```diff
@@ -90,7 +90,7 @@ case class PresentationConversionCompletedSysPubMsg(
     body:   PresentationConversionCompletedSysPubMsgBody
 ) extends StandardMsg
 case class PresentationConversionCompletedSysPubMsgBody(podId: String, messageKey: String, code: String,
-                                                        presentation: PresentationVO)
+                                                        presentation: PresentationVO, presentationConvertedName: String)
```

Comment:
Any reason for not setting `presentationConvertedName` inside `PresentationVO` class? (same place of the original name)